src/main/scala/tech/cryptonomic/conseil/Conseil.scala

```diff
@@ -35,8 +40,20 @@ object Conseil extends App with LazyLogging with EnableCORSDirectives with Conse
 
       val tezosDispatcher = system.dispatchers.lookup("akka.tezos-dispatcher")
       lazy val tezos = Tezos(tezosDispatcher)
-      lazy val platformDiscovery = PlatformDiscovery(platforms, caching)(tezosDispatcher)
-      lazy val data = Data(platforms)(tezosDispatcher)
+
+      // This part is a temporary middle ground between current implementation and moving code to use IO
+      implicit val contextShift: ContextShift[IO] = IO.contextShift(executionContext)
+      val attributesCache = MVar[IO].empty[Map[String, (Long, List[Attribute])]].unsafeRunSync()
```

Comment:
because in the end we would want to have all of the IO operations to compose, so probably we would have one big for comprehension which would help avoid `unsafeRunSync` call. At this point I had a choice to put `unsafeRunSync` in the  `PlatformDicoveryOp` or in here. I chose `Conseil` object to show the direction in which we should go.

Reply:
Awesome 👍